app/components/nav/LearnMoreLink.tsx

```diff
@@ -0,0 +1,14 @@
+import { Box } from '@mui/material'
+import Link from 'next/link'
+import ChevronRightIcon from '@mui/icons-material/ChevronRight'
+
+export const LearnMoreLink = () => {
+	return (
+		<Box sx={{color: '#0265DC', fontWeight: 500, letterSpacing: '.2px', display: 'flex', alignItems: 'center', padding: '8px 0'}}>
+			<Link href="#" style={{display: 'flex', alignItems: 'center'}}>
+				LEARN MORE
+				<ChevronRightIcon fill="#0265DC" />
+			</Link>
```

Comment:
for this I think we can just use button? that might help cut down on some of these styles. https://mui.com/material-ui/react-button/#buttons-with-icons-and-label

Reply:
I can add a TODO to fix this is a future branch or ticket